packages/superset-ui-legacy-preset-chart-nvd3/src/utils/isTruthy.js

```diff
@@ -1,8 +1,9 @@
 export default function isTruthy(obj) {
   if (typeof obj === 'boolean') {
     return obj;
-  } else if (typeof obj === 'string') {
-    return ['yes', 'y', 'true', 't', '1'].indexOf(obj.toLowerCase()) >= 0;
+  }
+  if (typeof obj === 'string') {
+    return ['yes', 'y', 'true', 't', '1'].includes(obj.toLowerCase());
```

Comment:
Not sure if the linter complained about this, but I kind of like `else if` stylistically, plus it's one less computation in some cases, but I won't lose any sleep over it :D

Reply:
The linter complains about it